sdk/eventgrid/azure-eventgrid/samples/publish_samples/publish_cloud_events_to_custom_topic_sample.py

```diff
@@ -0,0 +1,66 @@
+import sys
```

Comment:
all the samples must have a docstring in this template
https://github.com/Azure/azure-sdk-for-python/blob/master/sdk/storage/azure-storage-blob/samples/blob_samples_authentication.py#L10-L25

feel free to just log an issue for this if you are short on time

Reply:
logged an issue for this here: https://github.com/Azure/azure-sdk-for-python/issues/12933